doc/lua_api.txt

```diff
@@ -6494,21 +6494,17 @@ object you are working with still exists.
       amount.
     * `nil`: Disables override, defaulting to sunlight based on day-night cycle
 * `get_day_night_ratio()`: returns the ratio or nil if it isn't overridden
-* `set_local_animation(stand/idle, walk, dig, walk+dig, frame_speed=frame_speed)`:
-  set animation for player model in third person view
-
-      set_local_animation({x=0, y=79},  -- stand/idle animation key frames
-          {x=168, y=187},  -- walk animation key frames
-          {x=189, y=198},  -- dig animation key frames
-          {x=200, y=219},  -- walk+dig animation key frames
-          frame_speed=30)  -- animation frame speed
-* `get_local_animation()`: returns stand, walk, dig, dig+walk tables and
+* `set_local_animation(idle, walk, dig, walk_while_dig, frame_speed)`:
+  set animation for player model in third person view.
+    * Every animation equals to a `{x=starting frame, y=ending frame}` table.
+    * `frame_speed` sets the animations frame speed. Default is 30.
+* `get_local_animation()`: returns idle, walk, dig, dig+walk tables and
   `frame_speed`.
-* `set_eye_offset({x=0,y=0,z=0},{x=0,y=0,z=0})`: defines offset value for
-  camera per player.
+* `set_eye_offset(firstperson, thirdperson)`: defines offset value
+  for camera per player, as a table.
```

Comment:
I don't know what's with the weird need to mention tables everywhere.
If something is a position, it should be obvious that it's a `{x=0, y=0, z=0}` table:
https://github.com/minetest/minetest/blob/738f62421872c0be5fcd483ad4573e5d879e7418/doc/lua_api.txt#L1469-L1477

Reply:
In fact, mentioning 'table' is confusing, these are positions. Positions are technically tables but are not thought of that way, they are thought of as positions.